code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -353,7 +353,13 @@
 	playsound(target,'sound/effects/magic.ogg', 75, 1)
 	new /obj/effect/temp_visual/telekinesis(get_turf(target))
 	var/mob/living/carbon/xenomorph/patient = target
-	patient.heal_wounds(SHRIKE_CURE_HEAL_MULTIPLIER)
+
+	var/heal_amount = 1 + (patient.maxHealth * 0.03)
+	heal_amount += 0.008*patient.maxHealth*patient.recovery_aura
+	heal_amount *= SHRIKE_CURE_HEAL_MULTIPLIER
```

Comment:
Shrike heal used the xeno passive heal code. Since xenos passive blood loss code is in it and the proc's healing is now based on blood level, shrike heal would cause the patient to loose blood and the heal amount would be less at low blood level.

I can change where the blood drain code for passive healing is located, but this change makes shrike's heal self-contained like drone's.

Reply:
No I mean why is this split into 3 lines

Reply:
changed to one line